src/view/plugin/search/search.tsx

```diff
@@ -113,7 +113,6 @@ export class Search extends PluginBaseComponent<
             className('input'),
             className('search', 'input'),
           )}
-          value={this.store.state.keyword}
```

Comment:
Could you please add a test for this please? I'm wondering why we don't need the `keyword` value in the store